### ISSUE TITLE
Error when trying to format includes with empty lines (new issue with 2.0)

### ISSUE THREAD

**Issue by etiennehebert** [Wumpf/IncludeToolbox#33]
I get an error if I try to format the following:
```
#include "myproject/precomp.h"
#include "myproject/currentfile.h"

#include "myproject/stuff1.h"
#include "myproject/stuff2.h"
#include "myproject/stuff3.h"
```

I have the following options:
![image](https://cloud.githubusercontent.com/assets/8545740/26590995/3f642aac-452a-11e7-96cf-e54d294dd6b0.png)

If I put the option to remove the empty lines to true, it will work.
If I remove the empty line before running the formatter, it will also work.

Here is the error.

```
Unexpected Error: System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
   at System.Text.StringBuilder.set_Chars(Int32 index, Char value)
   at IncludeToolbox.Formatter.IncludeLineInfo.SetLineType(Type newLineType) in F:\Development\current_development\IncludeToolbox\IncludeToolBox\Formatter\IncludeLineInfo.cs:line 192
   at IncludeToolbox.Formatter.IncludeFormatter.FormatDelimiters(IEnumerable`1 lines, DelimiterMode delimiterMode) in F:\Development\current_development\IncludeToolbox\IncludeToolBox\Formatter\IncludeFormatter.cs:line 68
   at IncludeToolbox.Formatter.IncludeFormatter.FormatIncludes(String text, String documentPath, IEnumerable`1 includeDirectories, FormatterOptionsPage settings) in F:\Development\current_development\IncludeToolbox\IncludeToolBox\Formatter\IncludeFormatter.cs:line 200
   at IncludeToolbox.Commands.FormatIncludes.MenuItemCallback(Object sender, EventArgs e) in F:\Development\current_development\IncludeToolbox\IncludeToolBox\Commands\FormatIncludes.cs:line 80
   at IncludeToolbox.Commands.CommandBase`1.<SetupMenuCommand>b__1_0(Object sender, EventArgs e) in F:\Development\current_development\IncludeToolbox\IncludeToolBox\Commands\CommandBase.cs:line 39
```

Thanks!

**Comment by Wumpf** [comment]
Nice catch! Thx!
Got a repro here. Will fix asap and add a test for whatever caused that.

**Comment by Wumpf** [comment]
New version with fix is online (2.0.1)

**Comment by etiennehebert** [comment]
Fix works. Thanks!
